src/+otp/+pendulum/PendulumProblem.m

```diff
@@ -0,0 +1,79 @@
+classdef PendulumProblem < otp.Problem
+    properties (SetAccess = private)
+        RhsMass
+    end
+    
+    methods
+        function obj = PendulumProblem(timeSpan, y0, parameters)
+            % Constructs a problem
+            obj@otp.Problem('Pendulum', [], timeSpan, y0, parameters);
+        end
+        
+        function [x, y] = convert2Cartesian(obj, y, includeOrigin)
```

Comment:
includeOrigin need a sane default

Reply:
Also a function to get the angles from cartesian would be nice

Reply:
includeOrigin already has a default value of false.  A Cartesian to polar would probably have to be a static function since (x,y) points may not be consistent with Parameters.lengths.  For consistency and simplicity, I don't think it's worth adding.